core/src/main/scala/scalaz/Order.scala

```diff
@@ -83,22 +85,32 @@ object Order {
       override def iso: F <=> G = D
     }
 
-  ////
+  implicit val orderDecidableInstance: Decidable[Order] = new OrderDecidableInstance
 
-  implicit val orderInstance: Divisible[Order] = new Divisible[Order] {
-    override def contramap[A, B](r: Order[A])(f: B => A) = r.contramap(f)
+  private[scalaz] class OrderDecidableInstance extends Decidable[Order] with Divisible[Order] {
+    override def choose2[Z, A1, A2](order1: => Order[A1], order2: => Order[A2])(f: Z => ∨[A1, A2]): Order[Z] = order[Z] { (c1, c2) =>
```

Comment:
you're using `\/[A1, A2]` instead of `A1 \/ A2`... was that IntelliJ perhaps? It's definitely more idiomatic to use the infix notation.

Reply:
Changing it, IntelliJ not very clever....